### DISCUSSION TITLE
Supported data formats

### DISCUSSION THREAD

**Discussion by davedavemckay** [johntruckenbrodt/pyroSAR#290] [General]
Hello, I'm trying to run geocode on some _unzipped_ Sentinel-1 GRD scenes. So this might be a quick one! I'm running the example pyrosar.identify() code and get "data format not supported". I suspect this is either due to the datasets being cloud-optimized geoTIFF, or simply because they're ingested into the system I'm using as .SAFE folders, not zip files.

My source is ESA OData. I can download the original data, not the COG versions, if that's the issue, or zip downloaded datasets.

Thanks!

**Comment by davedavemckay** [comment]
It turns out I've answered my own question - downloading 'S1A_IW_GRDH_1SDV_20151021T063752_20151021T063817_008247_00B9CD_097D.SAFE' rather than 'S1A_IW_GRDH_1SDV_20151021T063752_20151021T063817_008247_00B9CD_2770_COG.SAFE', i.e., not using the COG, pyroSAR opens the dataset fine.
If anyone has any insights as to the differences here - would there be an advantage to using the COG version? - I'd be interested. It seems ESA have invested in generating them for the entire Sentinel-1 archive, so presumably they have a good reason.

**Comment by johntruckenbrodt** [comment]
Hi @davedavemckay. For my better understanding...what do you mean by ESA OData? As far as I know OData is just a web interface protocol and I don't know of any ESA-hosted data service. Are you referring to the new Copernicus Data Space Ecosystem (CDSE)? In this case I think the data comes from creoDIAS operated by CloudFerro. I am not sure about the reasoning behind the format conversion but could ask around.

**Reply by davedavemckay** [reply]
Hi @johntruckenbrodt Yes, sorry, I'm using OData to search for files, then I'm downloading them via the S3 object storage service offered by CDSE.
I noted this beacuse my previous experience with pyroSAR has been using Sentinel-1 data from ASF, just in case it differed in some way.
I use Sentinel-1 and Sentinel-2 data, so the new CDSE services are helping me simplify some things by having a lot of the data I need in one place 😄 
No worries about the reasons behind the COG format use - I'm up and running now with the original format, but was curious.

**Reply by johntruckenbrodt** [reply]
Thanks! Supporting CDSE data is definitely important. I have opened an issue to track this (https://github.com/johntruckenbrodt/pyroSAR/issues/291). Perhaps it just needs a little tweaking to recognize the new naming scheme but it could also be that SNAP does not support these files yet. SNAP 10 is supposed to be released this month and I am sure they will include any necessary fix.
